### PR TITLE
Details popup window 

### PR DESCRIPTION
**Pull request by chepkok3**
JavaScript array to store all of the information for all projects.
Store pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
Update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
Implement the popup window.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -16,3 +16,320 @@ navItem.forEach((list) => {
...
+</div>
+  </div>
+`;
+  projectContainer.innerHTML += eachDesktopProject;
+});
+
+const projectButtons = document.querySelectorAll('.text-see-project');
+
+const modalContent = (project) => {
+  const popup = `
```
Comment (Olamarx) [inline script.js:205]: - [ ] The popup should be centered as it is in the figma. This is a styling issue though.
### Your work
image

### Figma
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @chepkok3, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- GitHub flow ✔️
- popup functionalities ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chepkok3** [comment]
@Olamarx Hi, thanks for the review. I have tried to center the popup window to be similar to the Figma template but struggled with it. Can you suggest how I can do it without wasting more time?

File: script.js [hunk 2]
```diff
@@ -16,3 +16,320 @@ navItem.forEach((list) => {
...
+      ${tech}
+      </ul>
+    </div>
+    <div class="text-see-project">
+      <a class="see-project-button" id=${index} href="#">See Project</a>
+    </div>
+  </div>
+  </div>
+  </div>
+  <div>
```
Comment (mckent05) [inline script.js:119]: **OPTIONAL** - Your third project card seems to be causing a huge scroll on your page. Please fix this to help improve the UI of your portfolio. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your Project:
iiihih

**Review by mckent05 — APPROVED** [review]
Hi @chepkok3, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏
- You used the right workflow for this milestone 👍 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 2 above)
Comment (chepkok3) [reply script.js:119]: @mckent05 Thank you the optional suggestion. I will take it into consideration and make the necessary changes.🙏

**Pull request merged**